bin/govuk-docker

```diff
@@ -1,23 +1,44 @@
-#!/usr/bin/env ruby
+#!/bin/bash
```

Comment:
`/bin/bash` isn't portable, that should be `/usr/bin/env bash`

Reply:
Oops. I can't think why I made that change now - reverted!